### DISCUSSION TITLE
FAQ for large version jump upgrades

### DISCUSSION THREAD

**Discussion by mtinberg** [netbox-community/netbox#9671] [Ideas]
I noticed several people recently tasked with upgrading old (2.x) and very old (1.x) Netbox installations to the current release and thought that if one or more of those people who have recently done this upgrade path could put together some notes and a FAQ for how they did the upgrade, command history, problems they ran into, platform specific challenges, etc. that might be helpful to put in the main netbox docs for the next person who comes around asking the same questions.

No one probably has time for this, once you've upgraded you move on to another project, but I thought I'd just throw the idea out there in case I'm wrong about that.

**Comment by jeremystretch** [comment]
I've captured this in my notes for the ongoing docs refresh (#9838). Thanks!
